native/android/app/build.gradle

```diff
@@ -192,6 +197,11 @@ android {
     buildTypes {
         debug {
             signingConfig signingConfigs.debug
+            if (nativeArchitectures) {
+                ndk {
+                    abiFilters nativeArchitectures.split(',')
+                }
+            }
```

Comment:
Please also add the indentation somewhere later in this file to keep everything as close to the diff app as possible.